packages/server/src/migrations/functions/tests/tableSettings.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { App, Screen, ScreenProps } from "@budibase/types"
+
+const { db: dbCore } = require("@budibase/backend-core")
+const TestConfig = require("../../../tests/utilities/TestConfiguration")
+const migration = require("../tableSettings")
```

Comment:
Because this is a `spec.ts` you can use `import` here instead of require. We can then benefit from better type and build checking in this test for the imports as they will no longer resolve to `any`.